experimental/enable-metrics.yml

```diff
@@ -0,0 +1,46 @@
+- path: /instance_groups/name=bosh/properties/director/metrics_server?/enabled?
```

Comment:
You need not the subsequent question marks `?`. Only the most leftwards is necessary.

After you introduce one in the path, all the remaining nodes on the right might not exist.

This comment applies to all lines below where you have many question marks `?` in paths.